blocks/extensions/__init__.py

```diff
@@ -128,6 +128,24 @@ def on_interrupt(self):
         pass
 
 
+class CallbackName(str):
```

Comment:
In a way `Callback` class is just a function, because we do not need any additional functionality for them. I think it is confusing when a class named `Callback` does not even have `__call__` method.